CHANGELOG.md

```diff
@@ -2,6 +2,12 @@
 
 ## [UNRELEASED]
 
+## 1.4.12
+### Changed
+- ObjectForm fields now use Fomantic grid classes for fields layout (customizable as a prop)
+- Changed ObjectForm CSS to have a compact look
+- Added dash when no value exists in ObjectFormField
+
```

Comment:
Current published version is 1.4.9, I think both should be under 1.4.10 anyway before the release. Otherwise very nice job!